cmd/juju/model/destroy.go

```diff
@@ -157,18 +164,46 @@ func (c *destroyCommand) Run(ctx *cmd.Context) error {
 	defer api.Close()
 
 	// Attempt to destroy the model.
+	ctx.Infof("Destroying model")
 	err = api.DestroyModel(names.NewModelTag(modelDetails.ModelUUID))
 	if err != nil {
 		return c.handleError(errors.Annotate(err, "cannot destroy model"), modelName)
 	}
 
+	// Wait for model to be destroyed.
+	modelFound := newTimedModelFound(ctx, api, names.NewModelTag(modelDetails.ModelUUID), c.clock)
+	found := modelFound(0)
+	for found {
+		ctx.Infof("Waiting for model to be removed...")
```

Comment:
Can you please make this do the same thing as destroy-controller? i.e. like

    "Waiting on %d machines, %d applications"

The output is not very useful as it is.